src/wrapper/wrapper.ts

```diff
@@ -0,0 +1,37 @@
+import { UserData } from "./models/user";
+import { NotificationData } from "./models/notification";
+import { CountryData } from "./models/country";
+import { ErrorData, FEResponse } from "./models/common";
+import fetch from "node-fetch";
+
+export async function getUser(
+    apiKey: string
+): Promise<FEResponse<UserData | ErrorData>> {
```

Comment:
It would be preferable to have the wrapper throw errors instead of returning a union type like this